charts/scalardb-cluster/templates/scalardb-cluster/secret.yaml

```diff
@@ -1,13 +0,0 @@
-{{- if not .Values.scalardbCluster.existingSecret }}
```

Comment:
This file (Secret resource) is related to the deprecated `scalardbCluster.existingSecret` configuration.